iroha-cli/interactive/impl/interactive_query_cli.cpp

```diff
@@ -168,59 +173,67 @@ namespace iroha_cli {
           local_time_, creator_, counter_, account_id, asset_id);
     }
 
-    nonstd::optional<iroha::model::Pager> InteractiveQueryCli::parsePager(
+    iroha::model::Pager InteractiveQueryCli::parsePager(
```

Comment:
What is the reason to return the raw parameter without an optional wrapper?

Reply:
Answered in https://github.com/hyperledger/iroha/pull/830#discussion_r173097242